packages/core/src/index.html

```diff
@@ -5,10 +5,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Stencil Component Starter</title>
 
+    <link rel="stylesheet" href="/build/dso-toolkit.css">
     <script type="module" src="/build/dso-toolkit.esm.js"></script>
     <script nomodule src="/build/dso-toolkit.js"></script>
   </head>
   <body>
-
+    <dso-alert level="success">
```

Comment:
Hoe gaan we dit doen als iedereen zijn eigen componenten gaat bouwen? Is het wel nodig om dit in te checken en zo ja, wat wordt dan de volgorde van de componenten?

Reply:
Ik denk dat dit nog 'n beetje freestyle blijft. Binnenkort hang ik er een Storybook in.